core/src/test/java/com/google/cloud/sql/core/MockAdminApi.java

```diff
@@ -66,10 +66,10 @@ public class MockAdminApi {
 
   private static final Pattern CONNECT_SETTINGS_PATTERN =
       Pattern.compile(
-          ".*/sql/v1beta4/projects/(?<project>.*)/instances/(?<instance>.*)/connectSettings");
+          "(?<baseUrl>.*)sql/v1beta4/projects/(?<project>.*)/instances/(?<instance>.*)/connectSettings");
   private static final Pattern GENERATE_EPHEMERAL_CERT_PATTERN =
       Pattern.compile(
-          ".*/sql/v1beta4/projects/(?<project>.*)/instances/(?<instance>.*):generateEphemeralCert");
+          "(?<baseUrl>.*)sql/v1beta4/projects/(?<project>.*)/instances/(?<instance>.*):generateEphemeralCert");
```

Comment:
Nice. This is as good as an integration test, since it actually exercises the http client.